package.json

```diff
@@ -157,7 +157,7 @@
     "pm2-runtime": "./bin/pm2-runtime"
   },
   "dependencies": {
-    "async": "1.5",
+    "async": "^2.4.1",
```

Comment:
@soyuka If the tests are good, i don't see any reason not to merge it in (and they are mostly passing)